src/ansari/config.py

```diff
@@ -1,13 +1,12 @@
-import logging
 from functools import lru_cache
 from pathlib import Path
 from typing import Literal
 
 from pydantic import DirectoryPath, Field, PostgresDsn, SecretStr, field_validator
 from pydantic_settings import BaseSettings, SettingsConfigDict
 
-# Can't use get_logger() here due to circular import
-logger = logging.getLogger(__name__)
+# # Can't use get_logger() here due to circular import
```

Comment:
Unnecessary #?

Reply:
If you're saying we should use normal logging in `config.py` specifically instead of loguru, then sure, I can return lines 9,10 (in red) 

But I think _that_ won't be necessary, since we never logged anything in `config.py` to begin with, nor do I think we intend to log anything in the future, but obviously correct me if I've misunderstood you 🙌